src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        randomBall.setColor(colorSupplier.getRandomColor());
+        randomBall.setNumber(new Random().nextInt(100));
```

Comment:
Why creation of random color in separate variable, but creation of random number is not? Please make your code more consistent